src/operator/nn/fully_connected-inl.h

```diff
@@ -91,21 +140,14 @@ void FCForward(const OpContext &ctx, const FullyConnectedParam &param,
   CHECK_EQ(s->blas_handle_ownership_, Stream<xpu>::OwnHandle)
       << "Must init CuBLAS handle in stream";
 #endif  // __CUDACC__
-  const mxnet::TShape& ishape = in_data[fullc::kData].shape_;
```

Comment:
Please don't refactor this as it is not related to this PR.

Reply:
It's related I created a function that does that same thing because it's going to use in the two places. It's code duplication.